### DISCUSSION TITLE
Tesselation with fiber orientation - changed behaviour form v4.2 to v4.4.2

### DISCUSSION THREAD

**Discussion by michalkalkowski** [neperfepx/neper#423] [Q&A] [answered]
Hi,

I used neper to generate a 2D tesselation with fiber orientation, then rotated by a certain angle.
However, I noticed that the orientation output differs for the same command over the two versions. I did not find an explanation for the changed behaviour - can you explain why the change and how this should be understood?
command used:

`neper -T -dim 2 -id 1 -domain "square(60, 10)" -n 5000  -ori "fiber(0,0,1,0,1,0)" -transform "scale(1, 8, 1),rotate(0,0,1,30)" -oridescriptor "euler-bunge" -format "tess,ori" -o test_30_441
`
output orientations for v4.4.1:

-180.000000000000   90.000000000000   41.975192057725
 180.000000000000   90.000000000000 -117.505742464530
-180.000000000000   90.000000000000   50.562613184348
-180.000000000000   90.000000000000   71.730660197623
-180.000000000000   90.000000000000   41.725771745734
-180.000000000000   90.000000000000   70.710510813742
...

output orientations for v4.2:

-168.024807942275   90.000000000000   30.000000000000
  32.494257535470   90.000000000000   30.000000000000
-159.437386815652   90.000000000000   30.000000000000
-138.269339802377   90.000000000000   30.000000000000
-168.274228254266   90.000000000000   30.000000000000
-139.289489186258   90.000000000000   30.000000000000
 103.575792782177   90.000000000000   30.000000000000
...

What is neper doing? Why the change?
Thanks!
Michal

**Comment by rquey** [comment, marked as answer]
There has been a move from `fiber(<dirs_x>,<dirs_y>,<dirs_z>,<dirc_x>,<dirc_y>,<dirc_z>)` to `fiber(<dirc_x>,<dirc_y>,<dirc_z>,<dirs_x>,<dirs_y>,<dirs_z>)` between the two versions you mentionned.  You can find the new definition [here](https://neper.info/doc/neper_t.html#cmdoption-ori). Sorry this hasn't been reported properly in the `VERSIONS` file...

The reason is that the ordering is more natural, and that changes have to happen at some point...

**Comment by michalkalkowski** [comment]
Great, thanks! I did not notice the difference in the definition. All clear now.
